resources/artifact-ignores.properties

```
@@ -881,3 +880,10 @@ delphix@2.0.4
 
 # Dependency nexus-platform-api has a proprietary license, helpdesk#3742
 nexus-jenkins-plugin = https://github.com/jenkins-infra/helpdesk/issues/3742
+
+# Removing old Batch Task plugin releases
+batch-task@1.15
+batch-task@1.16
+batch-task@1.17
+batch-task@1.18
+batch-task@1.19
```

```suggestion
batch-task@1.15               # SECURITY-1025
batch-task@1.16               # SECURITY-1025
batch-task@1.17               # SECURITY-1025
batch-task@1.18               # SECURITY-1025
batch-task@1.19               # SECURITY-1025
```

To be consistent with others.